nemo_text_processing/text_normalization/verbalizers/verbalize.py

```diff
@@ -31,22 +32,27 @@ class VerbalizeFst(GraphFst):
     Composes other verbalizer grammars.
     For deployment, this grammar will be compiled and exported to OpenFst Finate State Archiv (FAR) File. 
     More details to deployment at NeMo/tools/text_processing_deployment.
+
+    Args:
+        deterministic: if True will provide a single transduction option,
+            for False multiple options (used for audio-based normalization)
     """
 
-    def __init__(self):
+    def __init__(self, deterministic: bool = True):
         super().__init__(name="verbalize", kind="verbalize")
-        decimal = DecimalFst()
-        decimal_graph = decimal.fst
-        cardinal = CardinalFst()
+        cardinal = CardinalFst(deterministic=deterministic)
         cardinal_graph = cardinal.fst
+        decimal = DecimalFst(deterministic=deterministic)
+        decimal_graph = decimal.fst
         ordinal = OrdinalFst()
         ordinal_graph = ordinal.fst
         telephone_graph = TelephoneFst().fst
         electronic_graph = ElectronicFst().fst
-        measure_graph = MeasureFst(decimal=decimal, cardinal=cardinal).fst
+        measure = MeasureFst(decimal=decimal, cardinal=cardinal)
```

Comment:
no deterministic? could we add determisitc do all classes? maybe GraphFST?

Reply:
added to GraphFst